src/commands/reputation/setReputation.ts

```diff
@@ -20,7 +20,9 @@ export const setReputation = async (msg: Message) => {
   if (!discordUser) return; // return if user not found
   if (discordUser.bot) return; // return if mention bot
 
-  const splittedMessage = msg.content.split(' ');
+  const splittedMessage = msg.content
+    .split(' ')
+    .filter((word) => word.length > 0);
```

Comment:
You can also do the following to get rid of all spaces in between words.

```suggestion
  const splittedMessage = msg.content.split(/\s+/);
```